tests/TestCase/Job/SendMailJobTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org/)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org/)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         0.1.9
+ * @license       https://opensource.org/licenses/MIT MIT License
+ */
+namespace Cake\Queue\Test\TestCase\Job;
+
+use Cake\Mailer\Mailer;
+use Cake\Mailer\Transport\DebugTransport;
+use Cake\Queue\Job\Message;
+use Cake\Queue\Job\SendMailJob;
+use Cake\Queue\Queue\Processor;
+use Cake\TestSuite\TestCase;
+use Enqueue\Null\NullConnectionFactory;
+use Enqueue\Null\NullMessage;
+
+class SendMailJobTest extends TestCase
+{
+    /**
+     * @var \Cake\Queue\Job\SendMailJob
+     */
+    protected $job;
+
+    /**
+     * @var \Cake\Mailer\Message
+     */
+    protected $message;
+
+    /**
+     * @inheritDoc
+     */
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->job = new SendMailJob();
+        $this->message = (new \Cake\Mailer\Message())
+            ->setFrom('from@example.com')
+            ->setTo('to@example.com')
+            ->setSubject('Sample Subject');
+    }
+
+    /**
+     * Test execute method
+     *
+     * @return void
+     */
+    public function testExecute()
+    {
+        $message = $this->createMessage(DebugTransport::class, [], $this->message);
+        $actual = $this->job->execute($message);
+        $this->assertSame(Processor::ACK, $actual);
```

Comment:
Are there any tests covering that the message was actually sent to the email transport?

Reply:
@markstory I have just improved the unit test to check send is called with correct parameters.